### DISCUSSION TITLE
Extension stopped working

### DISCUSSION THREAD

**Discussion by matthewjlevy** [mjisaak/skilling-champion-extension#8] [Q&A] [answered]
Hi MJ
It appears that the extension suddenly stopped working. I think I received an update to my browser (Edge - Version 88.0.692.0 (Official build) dev (64-bit))

**Comment by Jandev** [comment, marked as answer]
I was experiencing the same issue and navigated to this repo to file an issue.

Before doing so I uninstalled & re-installed the extension and it appears to be working again. I don't know why though.
![image](https://user-images.githubusercontent.com/462356/100755525-903ac080-33ec-11eb-8f2c-7eeb2457d983.png)

**Comment by matthewjlevy** [comment]
Thanks Jan, did the same and it works again.
